app/src/main/java/app/nameplaceholder/sensorsapp/ForegroundService.kt

```diff
@@ -21,6 +21,7 @@ class ForegroundService : Service() {
     override fun onDestroy() {
         super.onDestroy()
         isActiveChannel.offer(false)
+        stopForeground(true)
```

Comment:
> Leave the add/remove notification to the service. This will remove the notification when the service is stoped.

Isn't notification removed for you without this implicit call?

Reply:
I noticed only once that the notification was not updating more, but it was not removed. And I think, from an architecture point of view. The one that put the notification, should be the one to remove.

Reply:
Yup, I made this in a hour, might be some bugs :) But I see where problem is. `NotificationUpdater` migth add notification after it was removed. It should remove notification when service is not active.

But calling `stopForeground()` would not help to solve this case. After service is destroyed its notification is always removed automatically.